src/components/UserPage/UserPage.js

```diff
@@ -1,10 +1,135 @@
-import React, { Component } from 'react';
+import React, { Component, useEffect } from 'react';
 import { connect } from 'react-redux';
 import LogOutButton from '../LogOutButton/LogOutButton';
 import mapStoreToProps from '../../redux/mapStoreToProps';
 
+const mapStateToProps = (state, ownProps) => ({
+  dispatch: ownProps.dispatch,
+  history: ownProps.history,
+  user: state.user,
+  friends: state.friendsReducer,
+  decks: state.deck,
+})
+
+const UserPageFunc = ({
+  dispatch,
+  history,
+  user,
+  friends,
+  decks,
```

Comment:
Here I can de-structure the argument object. So I don't need to use an accessor everywhere. Like 'props.' in front of all my argument usage.